Plugins/Flow.Launcher.Plugin.BrowserBookmark/Views/CustomBrowserSetting.xaml

```diff
@@ -1,38 +1,192 @@
-            <ColumnDefinition Width="6*"/>
+            <ColumnDefinition Width="4*" />
+            <ColumnDefinition Width="6*" />
         </Grid.ColumnDefinitions>
-        <TextBlock Grid.Row="0" Grid.Column="0" Text="Browser Name" FontSize="15" 
-                   HorizontalAlignment="Left" VerticalAlignment="Center" Margin="80 0 0 0"/>
-        <TextBlock Grid.Row="1" Grid.Column="0" Text="Browser Data Directory Path" FontSize="15" 
-                   HorizontalAlignment="Left" VerticalAlignment="Center" Margin="80 0 0 0"/>
-        <TextBox Grid.Row="0" Grid.Column="1" Text="{Binding Name}"
-                   HorizontalAlignment="Left" VerticalAlignment="Center" Width="100" Height="30" Margin="50 0 0 0"/>
-        <TextBox Grid.Row="1" Grid.Column="1" Text="{Binding DataDirectoryPath}"
-                   HorizontalAlignment="Left" VerticalAlignment="Center" Width="200" Height="30" Margin="50 0 0 0"/>
-        <StackPanel HorizontalAlignment="Center" Grid.Row="2" Orientation="Horizontal" Grid.Column="1" Height="70">
-            <Button Name="btnConfirm" Content="Confirm" Margin="15" Click="ConfirmCancelEditCustomBrowser"/>
-            <Button Content="Cancel" Margin="15" Click="ConfirmCancelEditCustomBrowser"/>
+
+        <TextBlock
+            Grid.Row="0"
+            Grid.Column="0"
+            Margin="80,0,0,0"
+            HorizontalAlignment="Left"
+            VerticalAlignment="Center"
+            FontSize="15"
+            Text="{DynamicResource flowlauncher_plugin_browserbookmark_browserName}" />
+        <TextBlock
+            Grid.Row="1"
+            Grid.Column="0"
+            Margin="80,0,0,0"
+            HorizontalAlignment="Left"
+            VerticalAlignment="Center"
+            FontSize="15"
+            Text="{DynamicResource flowlauncher_plugin_browserbookmark_browserBookmarkDataDirectory}" />
+        <TextBox
+            Grid.Row="0"
+            Grid.Column="1"
+            Width="100"
+            Height="30"
+            Margin="50,0,0,0"
+            HorizontalAlignment="Left"
+            VerticalAlignment="Center"
+            Text="{Binding Name}" />
+        <TextBox
+            Grid.Row="1"
+            Grid.Column="1"
+            Width="200"
+            Height="30"
+            Margin="50,0,0,0"
+            HorizontalAlignment="Left"
+            VerticalAlignment="Center"
+            Text="{Binding DataDirectoryPath}" />
+        <StackPanel
+            Grid.Row="2"
+            Grid.Column="1"
+            Height="70"
+            HorizontalAlignment="Center"
+            Orientation="Horizontal">
+            <Button
+                Name="btnConfirm"
+                Margin="15"
+                Click="ConfirmCancelEditCustomBrowser"
+                Content="Confirm" />
+            <Button
+                Margin="15"
+                Click="ConfirmCancelEditCustomBrowser"
+                Content="Cancel" />
         </StackPanel>
+        -->
```

Comment:
@onesounds is there a reason we are keep this commented out code?

Reply:
I missed. removed.